test/integration/objectstore/_base.py

```diff
@@ -57,8 +57,14 @@ def setUp(self):
         self.dataset_populator = DatasetPopulator(self.galaxy_interactor)
 
 
+def get_files(directory):
+    for rel_directory, _, files in os.walk(directory):
+        for _file in files:
+            yield os.path.join(rel_directory, _file)
```

Comment:
```suggestion
        for file_ in files:
            yield os.path.join(rel_directory, file_)
```

See https://peps.python.org/pep-0008/#descriptive-naming-styles